### PR TITLE
Don't use named groups in RegExp

### PR DESCRIPTION
**Pull request by Velenir**
Firefox doesn't support named capturing groups :disappointed:

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr359--dexreact.review.gnosisdev.com

File: src/api/exchange/utils/decodeAuctionElements.ts [hunk 1]
```diff
@@ -27,7 +23,8 @@ export function decodeAuctionElements(bytes: string): AuctionElement[] {
   let index = 0 // order ID is given by position and it's not part of the encoded data
 
   while ((order = oneOrder.exec(bytes))) {
-    const {
+    const [
+      ,
```
Comment (alfetopito) [inline src/api/exchange/utils/decodeAuctionElements.ts]: Why this empty comma at the start?

**Comment by alfetopito** [comment]
I'll merge this to be able to play around with my PR https://github.com/gnosis/dex-react/pull/362

File: src/api/exchange/utils/decodeAuctionElements.ts (hunk 1 above)
Comment (Velenir) [reply src/api/exchange/utils/decodeAuctionElements.ts]: Destructuring with skipping
```js
// skipping 'A' and 'D'
const [,b,c,,e] = ['A','B','C','D','E']
```

**Pull request merged**